exercises/all-your-base/src/example.h

```diff
@@ -0,0 +1,12 @@
+#ifndef ALL_YOUR_BASE_H
+#define ALL_YOUR_BASE_H
+
+#include <stddef.h>
+#include <stdint.h>
+
+#define DIGITS_ARRAY_SIZE 64
+
+size_t rebase(int16_t digits[DIGITS_ARRAY_SIZE], int16_t fromBase,
```

Comment:
Both could be `int8_t`s. The array cannot be unsigned however as the test spec allows (but errors on) negative numbers